### DISCUSSION TITLE
Como utilizar RadioButton no React Native?

### DISCUSSION THREAD

**Discussion by PauloBragaDev** [react-hook-form/react-hook-form#8012] [General]
Estou com problemas para utilizar RadioButton no RN, na documentação não encontrei nenhum exemplo que utilize RadioButton no React Native.
Alguém tem algum exemplo disponível?

**Comment by bluebill1049** [comment]
Have a look at the radio button example for material UI, maybe it will help: 

https://codesandbox.io/s/react-hook-form-v7-controller-5h1q5
